### ISSUE TITLE
main.FATAL invalid argument

### ISSUE THREAD

**Issue by esetnik** [kahing/goofys#166]
I think i'm supplying the arguments correctly but goofys is returning an error with no info.

```
# goofys bucket-name /mnt/bucket-name
main.FATAL invalid argument
```

**Comment by kahing** [comment]
could you try to use `-f --debug_s3` to get more info?

**Comment by kahing** [comment]
it's also possible that syslog already contained the relevant output as well

**Comment by esetnik** [comment]
Yes I figured it out. I did not realize that the `-f` switch was required to run in the foreground. I would suggest that you change the default implementation so that a switch like `-d` is required to run in background. Daemonizing the process by default can lead to confusion because the user gets no feedback about what might be wrong. Especially when all the examples show the daemonized version. Thanks for the project it's really great work!
